src/settings-ui/Settings.UI/OOBE/Views/OobeShellPage.xaml.cs

```diff
@@ -195,7 +196,7 @@ private void NavigationView_SelectionChanged(Microsoft.UI.Xaml.Controls.Navigati
                 switch (selectedItem.Tag)
                 {
                     case "Overview":
-                        if (ExperimentationToggleSwitchEnabled)
+                        if (ExperimentationToggleSwitchEnabled && GPOWrapper.GetAllowExperimentationValue() != GpoRuleConfigured.Disabled)
```

Comment:
I would expect this condition in the `AllExperiments` code to not
- communicate with MS server if disabled.
- not have to check this in every place where we have experiments.

Reply:
At the current time, that involves spawning a Task and UI changes, so I think it makes sense to cut it right here. Also the experimentation code does some more logic like saving the status to keep it to the future, so I was thinking of keeping that code more isolated. It also requires testing in two different ways: with experimentation on and off, so I'd prefer to keep logic there as simple as possible, being the users of the code that end up having to understand if experimentation should be used or not.